src/backend/columnar/columnar_customscan.c

```diff
@@ -1100,6 +1100,29 @@ ParameterizationAsString(PlannerInfo *root, Relids paramRelids, StringInfo buf)
 }
 
 
+/*
+ * ContainsExecParams tests whether the node contains any exec params. The
+ * signature accepts an extra argument for use with expression_tree_walker.
+ */
+static bool
+ContainsExecParams(Node *node, void *notUsed)
```

Comment:
I think we can just remove notUsed from here, since we always pass NULL arg to walker

Reply:
Then the function signature won't match what expression_tree_walker expects. I think that might be undefined behavior in C, though it would probably work.

Reply:
oh got it, sorry I missed this.